content/news/2022-2023/2022-08-17-new-ccss-logo.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Our new Logo"
+date: 2022-08-17T00:00:00Z
+draft: false
+layout: overlaypost
+preview: "images/news/2022-2023/logo.jpg"
+background: "images/orientation2018-min.jpeg"
+excerpt: "This year, we're excited to introduce a brand new CCSS logo."
+hideBanner: false
+featured: true
+author: Matthew MacRae-Bovell
+url: /news/our-new-logo
+categories:
+- announcement
+---
+
+![New Logo](/images/news/2022-2023/logo.jpg)
+
+With our return to in-person activity this year, we really thought it was finally time to give the CCSS a new coat of paint. After quite a lot of variation, we're proud to present a symbol that we believe represents our program, community, and our mission as an organization. This will become our official logo at the beginning of the fall semester and we will begin slowly transitioning over to this logo over the next few weeks.
+ 
+<br>
+
+# Why we needed a new logo
+
+![Logo Change](/images/news/2022-2023/logo_change.jpg)
+
+Although we loved our previous logo very dearly, there were some objective issues with its design.
```

Comment:
I'm not sure I like saying "objective issues", but I'm not too set on changing it.